tools/local-network-setup/Readme.md

```diff
@@ -0,0 +1,88 @@
+Local Network Setup Tool
+========================
+
+#### Run a single command to create an entire testing enviroment for ot-node development.
+
+The Local Network Setup tool will start a local blockchain, deploy the required smart contracts, set up the configuration files for the nodes and start the nodes in separate windows.
+From there you're ready to send API calls to your local nodes and test new features on the ot-node without worrying about funds, servers or network connectivity issues.
+
+**Note: This tool is an internal tool used by the OriginTrail team and thus is developed for our workflow, meaning that it currently only supports MacOS**, but we encourage you to adapt it for your workflow as well.
+
+Quick Start
+===========
+
+## Prerequisites
+
+* You need to have arangodb installed and running on your machine. You can find instructions on the [ArangoDB website](https://www.arangodb.com/docs/stable/getting-started-installation.html)
+
+* You should have ot-node dependencies installed with the `npm install` command
+
+## How to start
+
+From the ot-node directory, run the below command
+
+```bash
+npm run tools:lns:start
+```
+
+Usage
+=====
+
+## Specifying the number of nodes
+
+The LNS tool deploys 4 nodes, each connected to two blockchain implementations which are running on a local ganache process.
```

Comment:
[minor] I would just mention here that there is a limit of 10 nodes, Everything else looks good, i've tested it locally and it works :-)